### ISSUE TITLE
Choose player to be launched on start

### ISSUE THREAD

**Issue by GoogleCodeExporter** [memeco/osd-lyrics#130]
```
What steps will reproduce the problem?
请列出导致问题的操作
1. Run OSDLyrics without any player running
2. Nothing happens

What's the output in terminal (run with command `osd-lyrics -d debug')?
请给出在终端下运行的输出（使用“osd-lyrics -d
debug”来运行）
No Player running

What's the player and which version you are using?
您使用的播放器是什么？它的版本号是多少？
Any player

What's version of OSD Lyrics?
OSD Lyrics的版本是多少？
PPA 0.3.20100604

What's your Linux distro?
您使用哪种发行版？
Ubuntu 10.04

Maybe the best way to solve this ugly "bug" is:
- QUICK FIX: Gtk error message "No compatible player running found".
- Best behaviour: Searching on /usr/bin for compatible players and let the user 
choose which to run. If no compatible players found ask to install a compatible 
one showing a list of compatible players.

```

Original issue reported on code.google.com by `orfe...@gmail.com` on 11 Sep 2010 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
It's not a bug. We designed it this way. We do not want OSD lyrics to run 
without any player.
```

Original comment by `sarlmola...@gmail.com` on 11 Sep 2010 at 2:04
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Well, I would accept this as a feature request.
In fact, I planned to implement this feature 2 months ago :), but I'm too lazy 
to implement it :(
```

Original comment by `tigerso...@gmail.com` on 11 Sep 2010 at 4:11
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tigerso...@gmail.com` on 11 Sep 2010 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
Implemented. Will be released in 0.4.1
```

Original comment by `tigerso...@gmail.com` on 23 Jun 2011 at 2:32
- Changed state: **Fixed**
- Added labels: **Type-FeatureRequest**
- Removed labels: **Type-Bug**
